libsolidity/interface/CompilerStack.cpp

```diff
@@ -556,7 +556,7 @@ Json::Value CompilerStack::assemblyJSON(string const& _contractName, StringMap _
 
```

Comment:
The function signature here has been wrong all the time, it should be `const& _sourceCodes`.